models.py

```diff
@@ -165,6 +165,19 @@
 MINOR_EXISTING_CHANGES = 4
 EXTREMELY_SMALL_CHANGE = 5
 
+# Status for security and privacy reviews.
+REVIEW_PENDING = 1
+REVIEW_IN_PROGRESS = 2
+REVIEW_COMPLETED = 3
+REVIEW_NA = 4
+
+REVIEW_STATUS_CHOICES = {
+    REVIEW_PENDING: 'Pending',
+    REVIEW_IN_PROGRESS: 'In progress',
```

Comment:
I think this should be "open issues" instead of "in progress".  Reviews are typically quick; they can stay "pending" until issues have been filed or all issues have been addressed (if there aren't any).

Reply:
Done